android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/TiListView.java

```diff
@@ -895,8 +895,11 @@ public View layoutHeaderOrFooterView(TiViewProxy viewProxy)
 
 	protected void processSections(Object[] sections)
 	{
-
+		for (ListSectionProxy listSectionProxy : this.sections) {
+			listSectionProxy.releaseViews();
+		}
```

Comment:
**Question:**
Calling `releaseViews()` will release the section. Will this put the JavaScript `ListSection` object in a bad state and prevent the developer from re-adding the section to the `ListView`?